modules/web-mercator/src/index.js

```diff
@@ -1,4 +1,5 @@
 // Classic web-mercator-project
+export {default} from './web-mercator-viewport';
```

Comment:
The same entry is needed in the `d.ts` file as well, I believe.

Reply:
d.ts still has this line.